easybuild/easyconfigs/m/multitail/multitail-6.2.1-intel-2015a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'multitail'
+version = '6.2.1'
+
+homepage = 'http://www.vanheusden.com/multitail/'
+description = """MultiTail allows you to monitor logfiles and command
+output in multiple windows in a terminal, colorize, filter and merge."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = [SOURCE_TGZ]
+source_urls = ['http://www.vanheusden.com/multitail/']
+
+patches = ['multitail-6-fix_makefile.patch']
+
+files_to_copy = [
+    (['multitail'], 'bin'),
+    (['multitail.conf'], 'etc'),
+]
+
+dependencies = [("ncurses", "5.9")]
+
+prebuildopts = 'DESTDIR=%(installdir)s '
+
+sanity_check_paths = {
+    'files': ['bin/multitail'],
+    'dirs': ["."]
```

Comment:
it's there intentionally in the template, to make the sanity check pass if you're doing `eb --try-software=foo,1.2.3 ...`